code/game/gamemodes/objective.dm

```diff
@@ -421,6 +423,55 @@ GLOBAL_LIST(admin_objective_list) //Prefilled admin assignable objective list
 			return ..()
 	return SSshuttle.emergency.is_hijacked() || ..()
 
+/datum/objective/gimmick
+	name = "gimmick"
+	martyr_compatible = TRUE
+
+/datum/objective/gimmick/update_explanation_text()
+	var/selected_department = pick(list( //Select a department for department-based objectives
+		"Research & Development",
+		"Engineering",
+		"Security",
+		"Medical",
+		"Service",
+		"Cargo",
+		"Command"
```

Comment:
aren't those defines somewhere?

Reply:
similar values are defined in two places, for radio channels and for playtime tracking, but no, not really.

Reply:
Well then you could consolidate the defines and move them to a new file

Reply:
fixed